runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -734,6 +737,28 @@ boolean isIgnored(String name)
 		return ignoredItems.contains(name);
 	}
 
+	void toggleEvent(String name, boolean ignore)
+	{
+		final Set<String> ignoredSet = new HashSet<>(ignoredEvents);
+
+		if (ignore)
+		{
+			ignoredSet.add(name);
+		}
+		else
+		{
+			ignoredSet.remove(name);
+		}
+
+		config.setIgnoredEvents(Text.toCSV(ignoredSet));
```

Comment:
If the config item for this is not actually going to be hidden, this will cause it to reorder itself randomly which seems not great

Reply:
How would I prevent this re-ordering itself? 

This method is a copy/paste of the existing `toggleItem` method which is also not hidden in the config panel, so it must have the same issue.